docs/dev_setup.rst

```diff
@@ -11,12 +11,58 @@ This section describes how someone wanting to contribute to the libCellML projec
 Overview
 ========
 
-The libCellML codebase is hosted on GitHub and therefore Git is used to track changes in the codebase.  Before you begin you will need to have a few prerequisites satisfied:
+The libCellML codebase is hosted on GitHub and therefore Git is used to track changes in the codebase.  Before you begin you will need to have a few pre-requisites satisfied:
 
-#. GitHub user account (for the rest of this document we will call our user *andre*)
-#. Git installed
+#. GitHub user account (for the rest of this document we will call our user *andre*).
+#. Git installed.
+#. CMake installed.
+#. Toolchain for building software installed (dependent on the operating system).
```

Comment:
#. GitHub user account (for the rest of this document we will call our user *andre*).
#. Git.
#. CMake.
#. Toolchain (dependent on the operating system).